explainer.md

```diff
@@ -732,6 +639,143 @@ Many thanks for valuable feedback, advice, and tools from:
+such as a landmark element which helps a user navigate sections of the page,
+or they may interact with interactive nodes,
+such as a button.
+In both of these cases,
+the node will usually need to have a **label** (often referred to as a **name**)
+to indicate the node's purpose in context.
+For example, a button may have a label of "Ok" or "Menu".
+
+Accessibility nodes may also have other properties,
+such as the current **value**
+(e.g. `"10"` for a range, or `"Jane"` for a text input),
+or **state** information
+(e.g. `"checked"` for a checkbox, or `"focused"`).
+
+Interactive accessibility nodes may also have certain **actions** which may be performed on them.
+For example, a button may expose a `"press"` action, and a slider may expose
+`"increment"` and `"decrement"` actions.
+
+These properties and actions are referred to as the *semantics* of a node.
+Each accessibility API expresses these concepts slightly differently,
+but they are all conceptually similar.
+
+##  Background: DOM tree, accessibility tree and platform accessibility APIs
+
+The web has rich support for making applications accessible,
+but only via a *declarative* API.
+
+The DOM tree is translated, in parallel,
+into the primary, visual representation of the page,
+and the accessibility tree,
+which is in turn accessed via one or more *platform-specific* accessibility APIs.
+
+![HTML translated into DOM tree translated into visual UI and accessibility tree](images/DOM-a11y-tree.png)
+
+Some browsers support multiple accessibility APIs across different platforms,
+while others are specific to one accessibility API.
+However, any browser that supports at least one native accessibility API
+has some mechanism for exposing a tree structure of semantic information.
+We refer to that mechanism, regardless of implementation details,
+as the **accessibility tree** for the purposes of this API.
+
+### Mapping native HTML to the accessibility tree
+
+Native HTML elements are implicitly mapped to accessibility APIs.
+For example, an  `<img>` element will automatically be mapped
+to an accessibility node with a role of `"image"`
+and a label based on the `alt` attribute (if present).
+
+![<img> node translated into an image on the page and an accessibility node](images/a11y-node-img.png)
+
+
+### ARIA
+
+Alternatively, [ARIA](https://www.w3.org/TR/wai-aria-1.1/)
+allows developers to annotate elements with attributes to override
+the default role and semantic properties of an element -
+but not to expose any accessible actions.
+
+![<div role=checkbox aria-checked=true> translated into a visual presentation and a DOM node](images/a11y-node-ARIA.png)
+
+In either case there's a one-to-one correspondence
+between a DOM node and a node in the accessibility tree,
+and there is minimal fine-grained control over the semantics of the corresponding accessibility node.
+
 ## Appendix: `AccessibleNode` naming
 
 We have chosen the name `AccessibleNode` for the class representing one
```

Comment:
This bit about accessibleNode seems like it may be out of date?

Reply:
Tweaked.